### PR TITLE
Module - 1: Capstone Project - WOS Concert Webpage

### PR DESCRIPTION
**Pull request by tan12082001**
Microverse Module - 1, week - 5: Capstone project
Project: HTML/CSS & JavaScript capstone project - Conference page.
As it is up to the choice of Micronauts to select the topic of the webpage, I choose to do a Concert Page.
Project Requirements achieved:
*  Personalize the content of your page
*  following the design guidelines for font sizes, colours, font face and layout.
* Build two web pages:

1.  Home Page
2.  About page

* To have a Responsive webpage with breakpoints for the mobile version and desktop version.
* home page should have a link in the menu to the About page
* logo in the header links to the home page.
* Mobile version should have a hamburger button and a popup of navigation elements.
* The section "Featured Artists" should be created dynamically in JavaScript.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+  width: 150px;
+  height: 51px;
+  margin-right: 10px;
+}
+
+.mobile-logo-img {
+  width: 100%;
+  height: 100%;
+}
+
```
Comment (tugsem) [inline main.css]: - [x] Kindly remove the logo from the mobile version of the app to match the design.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+  color: #333;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
```
Comment (tugsem) [inline main.css:29]: - [x] Kindly set your header's background color to transparent, remove the logo, and add weight to your hamburger icon to match this design:
yours: 
image
given:
image

File: main.css [hunk 3]
```diff
@@ -0,0 +1,746 @@
...
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-family: var(--main-font-family);
+  color: #333;
```
Comment (tugsem) [inline main.css]: - [x] Kindly set your font color to this value as in the given design which is #272A31: 
image

File: about.css [hunk 4]
```diff
@@ -0,0 +1,560 @@
...
+  transform: translateX(0);
+}
+
+.head-section {
+  background: url('./media/backgrounds/aboutMobileBackground\ \(3\).png') no-repeat top;
+  text-align: center;
+  padding-top: 51px;
+  width: 90%;
+  margin: 0 auto;
+}
```
Comment (tugsem) [inline about.css]: - [x] Kindly remove the margins at the sides, the background image should cover the page and the header as in the given design. :
yours: 
image
given: 
image

File: about.css [hunk 5]
```diff
@@ -0,0 +1,560 @@
...
+  margin: 17px 0.75em 0 0.75em;
+  width: calc(100% - 2.5em);
+  padding: 1em 0.5em;
+  font-size: 14px;
+  font-weight: bold;
+  height: auto;
+  text-align: justify;
+  background-color: #fff;
+  line-height: 150%;
+}
```
Comment (tugsem) [inline about.css]: - [x] Kindly center this div horizontally and set its border color to light grey to match the given design I attached the images above.

File: about.css [hunk 6]
```diff
@@ -0,0 +1,560 @@
...
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-family: var(--main-font-family);
+  color: #333;
```
Comment (tugsem) [inline about.css]: - [x] The font color should be the same as in the main.css which should be #272A31 to match the given design.

File: main.css [hunk 7]
```diff
@@ -0,0 +1,746 @@
...
+.headline-h2 {
+  color: var(--orange-shade-color);
+}
+
+.headline-h1 {
+  font-size: 2em;
+  background-image: url('./media/backgrounds/imagefortext.jpg');
+  background-size: cover;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
```
Comment (tugsem) [inline main.css:168]: - [x] On the main page, these main titles should be aligned to the left. Also, your h1 title should be written in the uppercase. Kindly fix it to match the given design: 
yours: 
image
given:
image

File: main.css [hunk 8]
```diff
@@ -0,0 +1,746 @@
...
+    background: #3e3e3e;
+  }
+
+  .header-bar-list {
+    display: flex;
+    gap: 24px;
+    list-style: none;
+    color: #fff;
+    margin: 6px;
+  }
```
Comment (tugsem) [inline main.css]: - [x] Please add spacing to the right of this div to match the given design layout:
yours:
image
given: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @tan12082001 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️ Speakers are created dynamically
✔️content is personalized
✔️style almost looks identical
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+            <div class="mobile-nav">
+                <nav class="with-hamburger">
+                    <div class="slidediv">
+                        <ul class="olist">
+                            <a class="olist-a" href='about.html'><li>About</li></a>
+                            <a class="olist-a" href="#main-program"><li>Program</li></a>
+                            <a class="olist-a" href="#join-event-button"><li>Join</li></a>
+                            <a class="olist-a" href="#partners"><li>Sponsor</li></a>
+                            <a class="olist-a" href="#"><li>News</li></a>
+                        </ul>
```
Comment (mckent05) [inline index.html:21]: **OPTIONAL** - You did well wrapping this section in a `ul` tag, but to help improve the semantics of your page, please wrap each `a` tag in a corresponding `li` tag. Please fix this for both pages 🙏

File: index.html [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+                            <li><a href='#main-program'>Program</a></li>
+                            <li><a href='#join-event-button'>Join</a></li>
+                            <li><a href='#partners'>Sponser</a></li>
+                            <li><a href=''>News</a></li>
+                        </ul>
+                    </div>
+                    <div class="tour-button">
+                        <button class="wos-tour-button">WOS Tour</button>
+                    </div>
+                </nav>
```
Comment (mckent05) [inline index.html:58]: **OPTIONAL** - It is not advisable to have two `nav` tags on one HTML page, I suggest you have just one nav tag, then use CSS to customize the nav for either the mobile or desktop version. Please fix this for both pages 🙏

File: index.html [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+            </div>
+        </header>
+        <main class="maincontent">
+            <section class="head-section">
+                <div class="headings">
+                    <h2 class="headline-h2">"Hello! Sharing world"</h2>
+                <h1 class="headline-h1">White Orange Sky Summer Concert 2023</h1>
```
Comment (mckent05) [inline index.html:65]: **OPTIONAL** - You have done well so far, but to help improve the semantics of your page, it is advisable to let your `hx` tag follow an ascending hierarchy, i.e use an `h2`tag after an `h1` tag, use an `h3` tag after an `h2` tag e.t.c. Please fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @tan12082001 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- You have done well completing your first capstone project, keep going 🚀 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**